tests/ipc_sock.test

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+#
+# Run the IPC tests under the stat wrapper,
+# this simulates /etc/libqb/use-filesystem-sockets existing
+# so we can test both options without breaking other things
+# that might be running on this system
+#
+if [ "`uname -s`" = "Linux" ]
```

Comment:
General comment here, also spotted below in other test scripts. It´s best to avoid the `command.. ` format and use instead "$(command..)". When using `` the command is forked in another shell that resets the envvar to default. This could cause unexpected behavior. Similar for `pwd` etc. forking also uses more resources. With $(cmd) the command is executed within the same shell.